packages/auth/src/access/types.ts

```diff
@@ -8,15 +7,15 @@ export enum AccessType {
 export enum Action {
   Create = 'create',
   Read = 'read',
+  ReadAll = 'read-all',
   List = 'list',
+  ListAll = 'list-all',
   Complete = 'complete'
 }
 
 interface BaseAccessRequest {
   actions: Action[]
-  locations?: string[]
   identifier?: string
-  interval?: string
```

Comment:
Should `interval` be moved to the `IncomingPaymentRequest` and `OutgoingPaymentRequest` types?

Reply:
It should be added to:
https://github.com/interledger/rafiki/blob/48cdfa261ce6857810cd1e5a080c411788db5da2/packages/auth/src/access/types.ts#L120-L124
https://github.com/interledger/open-payments/blob/main/openapi/schemas.yaml#L151-L163
I'll try to include that without scope-creeping too much more. I expect a lot of outstanding cleanup will be addressed with:
- https://github.com/interledger/rafiki/issues/662